parser.y

```diff
@@ -2429,19 +2429,31 @@ DropViewStmt:
 DropUserStmt:
 	"DROP" "USER" UsernameList
 	{
-		$$ = &ast.DropUserStmt{IfExists: false, UserList: $3.([]*auth.UserIdentity)}
+		$$ = &ast.DropUserStmt{IsDropRole: false, IfExists: false, UserList: $3.([]*auth.UserIdentity)}
```

Comment:
how about using a new `DropRoleStmt`?

Reply:
actually there is no difference between `DROP USER` and `DROP ROLE`. I think `IsDropRole` can be removed.